payments/serializers.py

```diff
@@ -3,3 +3,7 @@
 
 class StripeCheckoutSerializer(serializers.Serializer):
     redirect_url = serializers.CharField()
+
+
+class StripeCancelSerializer(serializers.Serializer):
+    pass
```

Comment:
```suggestion
```

Reply:
si no se pone, spectavular da error de que get selializer is none